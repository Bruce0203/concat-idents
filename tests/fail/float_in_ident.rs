concat_idents::concat_idents!(fn_name = foo, 1.0 {
    fn fn_name() {
        /* floats are not allowed in idents */
    }
});

fn main() {}
