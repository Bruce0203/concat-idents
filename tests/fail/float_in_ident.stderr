error: Identifiers cannot contain floats
 --> $DIR/float_in_ident.rs:1:46
  |
1 | concat_idents::concat_idents!(fn_name = foo, 1.0 {
  |                                              ^^^
